src/e2e-test/resources/pluginDataCyAttributes.properties

```
@@ -2,6 +2,8 @@ projectId=project
 datasetProjectId=datasetProject
 referenceName=referenceName
 table=table
+allowSchema=switch-allowFlexibleSchema
+SplitField=splitField
```

nit: please use camel case
```suggestion
splitField=splitField
```